x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/utils/MlTaskParams.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.core.ml.utils;
+
+public interface MlTaskParams {
+
+    String getTaskId();
```

Comment:
`getTaskId()` seems like it could be a confusing name here, because it returns neither the local task ID that the tasks API returns, nor the persistent task ID that persistent tasks uses.  It returns the ID used within ML.  Maybe `getMlId()` would be less likely to cause confusion in the future.